indico/modules/events/papers/templates/management/assignment.html

```diff
@@ -0,0 +1,95 @@
+{% extends 'layout/full_content_base.html' %}
+{% from 'events/papers/management/_paper_assignment_list.html' import render_paper_assignment_list %}
+{% from 'events/management/_lists.html' import render_displayed_entries_fragment %}
+
+{% block title %}
+    {% trans %}Paper Assignment{% endtrans %}
```

Comment:
The title should be the same in all paper-related pages. In this case, `Call for Papers` I believe. 
This we can move to the `subtitle` block.